parm/config/gefs/config.stage_ic

```diff
@@ -7,32 +7,27 @@ echo "BEGIN: config.stage_ic"
 # Get task specific resources
 source "${EXPDIR}/config.resources" stage_ic
 
-case "${CASE}" in
-  "C384")
-    export CPL_ATMIC=""
-    export CPL_ICEIC=""
-    export CPL_OCNIC=""
-    export CPL_WAVIC=""
-    export CPL_MEDIC=""
-    ;;
-  "C96")
-    export CPL_ATMIC=""
-    export CPL_ICEIC=""
-    export CPL_OCNIC=""
-    export CPL_WAVIC=""
-    export CPL_MEDIC=""
-    ;;
-  "C48")
-    export CPL_ATMIC="gefs_test"
-    export CPL_ICEIC="gefs_test"
-    export CPL_OCNIC="gefs_test"
-    export CPL_WAVIC="gefs_test"
-    export CPL_MEDIC="gefs_test"
-    ;;
-  *)
-    echo "FATAL ERROR Unrecognized resolution: ${CASE}"
-    exit 1
-    ;;
-esac
+export ICSDIR="@ICSDIR@" # User provided ICSDIR; blank if not provided
+export BASE_IC="@BASE_IC@" # Platform home for staged ICs
+
+export STAGE_IC_YAML_TMPL="${PARMgfs}/stage/stage.yaml.j2"
+
+# Set ICSDIR
+
+if [[ -z "${ICSDIR}" ]] ; then
```

Comment:
If User does not provide and `ICSDIR` remains` @ICSDIR@`, the if statement will be executed.
What if ICSDIR and BASE_IC are both not provided.
Why do we need both?

Reply:
`ICSDIR` will either be a path provided by the user or it will be empty (`''`):
```
parser.add_argument('--icsdir', help='full path to user initial condition directory', type=str, required=False, default='')
```
For most of my testing I do not provide an `ICSDIR` and the variable in the configs ends up empty:
```
export ICSDIR="" # User provided ICSDIR; blank if not provided
```
The check on line 16 of `config.stage_ic` then gets invoked. Here is an example from a `gdasstage_ic` job where I did not provide an `ICSDIR`:
```
+++ config.stage_ic[10]: export ICSDIR=
+++ config.stage_ic[10]: ICSDIR=
+++ config.stage_ic[11]: export BASE_IC=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
+++ config.stage_ic[11]: BASE_IC=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
+++ config.stage_ic[13]: export STAGE_IC_YAML_TMPL=/scratch1/NCEPDEV/global/Kate.Friedman/git/feature-issue_2475/parm/stage/stage.yaml.j2
+++ config.stage_ic[13]: STAGE_IC_YAML_TMPL=/scratch1/NCEPDEV/global/Kate.Friedman/git/feature-issue_2475/parm/stage/stage.yaml.j2
+++ config.stage_ic[16]: [[ -z '' ]]
+++ config.stage_ic[18]: ic_ver=20240610
+++ config.stage_ic[20]: ((  NMEM_ENS > 0  ))
+++ config.stage_ic[21]: ensic=C48
+++ config.stage_ic[24]: [[ NO == \Y\E\S ]]
+++ config.stage_ic[28]: export ICSDIR=/scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C48/20240610
+++ config.stage_ic[28]: ICSDIR=/scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C48/20240610
+++ config.stage_ic[32]: echo 'END: config.stage_ic'
```

`BASE_IC` comes from the host file and will (should) always be set. It is used to build `ICSDIR` when `ICSDIR` isn't provided by the user or otherwise set:
```
export ICSDIR="${BASE_IC}/${CASE}${ensic:-}${ocnic:-}/${ic_ver}"
```